Dockerfile-ubuntu

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.17 AS build-env
```

Comment:
The only difference between this Dockerfile and the other one already present is that this builds Desmos starting from a Ubuntu image rather than an Alpine one. This is going to be used inside `make build-linux` from now on, so we can link all the libraries we need (eg. `cosmwasm`) without much problems